tests/samplers_tests/tpe_tests/test_sampler.py

```diff
@@ -5,11 +5,13 @@
 from typing import Union
 from unittest.mock import Mock
 from unittest.mock import patch
+import warnings
```

Comment:
I don't think we explicitly silence `ExperimentalWarning`s in other tests. Mostly just curious but how come we should do it here? Also I think it's a good practice to make the filter scoped/temporary with `with warnings.catch_warnings():`